app/jobs/renew_checkout_job.rb

```diff
@@ -31,29 +22,34 @@ def perform(resource:, item_key:, session_token:)
       )
 
       Redis.current.set("renewal_#{item_key}", {
-        item_key: item_key,
+        id: item_key,
         result: :success,
-        renewal_count: checkout.renewal_count,
-        due_date: due_date,
-        status: checkout.status_human
+        response: {
+          renewal_count: checkout.renewal_count,
+          due_date: due_date,
+          status: checkout.status_human
+        }
       }.to_json)
     else
-      Sidekiq.logger.error("renewal_#{item_key}: #{response}")
+
+      processed_error = SirsiResponse::Error.new(error_message_raw: JSON.parse(response.body),
+                                                 symphony_client: symphony_client,
+                                                 key: item_key,
+                                                 session_token: session_token,
+                                                 bib_type: :checkout)
+
+      Sidekiq.logger.error("renewal_#{item_key}: #{processed_error.log}")
 
       Redis.current.set("renewal_#{item_key}", {
-        item_key: item_key,
+        id: item_key,
         result: :failure,
-        error_message: renewal_error_message(response)
+        response: {
+          renewal_count: 'Error',
+          due_date: 'Error',
+          status: 'Error'
```

Comment:
What about rather keeping the previous values? It just looks there are too many `Error`s on the page when there many renewals attempted :) Just a thought..

Reply:
Yeah, I thought about that too. Probably better? Maybe like providing the current values but with a warning sign thingy like a exclamation point in a triangle next to it? Idk. Good feedback. We can ask Ruth too.

Reply:
I like the idea of current values with a warning sign icon next to it, yes let's check with Ruth on this.

Reply:
fyi asked for Ruth's feedback here https://github.com/psu-libraries/myaccount/issues/179#issuecomment-653579690